### PR TITLE
Adding municipalities, naics

### PR DESCRIPTION
**Pull request by MrMaksimize**

### CODE REVIEW

**Comment by MrMaksimize** [comment]
@ainsleywagon review por favor :) ?

File: lib/OptionsList.js [hunk 1]
```diff
@@ -254,6 +246,9 @@ function choicesOverrideList(formName) {
       age: {
         '0': null
       },
+      eligibleBusinessLocation: {
```
Comment (ainsleywagon) [inline lib/OptionsList.js:249]: Hey hey. Totally see what you're doing in this PR - but not sure why this is here, when you've added it above. Does it take away what you have on line 156?

**Comment by ainsleywagon** [comment]
@MrMaksimize Looks good to merge!

File: lib/OptionsList.js (hunk 1 above)
Comment (MrMaksimize) [inline lib/OptionsList.js:249]: Oh heh yeah.  That's my "override" system.  So we don't have to define two lists of options for the same form, it basically overrides to "remove" an option in a specific form.

**Comment by MrMaksimize** [comment]
@ainsleywagon click da button! :whale:

**Pull request merged**